app/components/preprint-status-banner/component.js

```diff
@@ -251,10 +246,23 @@ export default Component.extend({
         },
     },
 
-    _handleActions(actions) {
-        if (actions.length) {
+    _getRecentAction(actions) {
```

Comment:
Should I just copy his code?

Reply:
If you want to. Or write it however you like -- since this is a hotfix, it'll get in first and @sheriefvt can use or modify it however it makes sense.

Reply:
Ah, I see what you mean.